javascript/src/components/grok-patterns/EditPatternModal.jsx

```diff
@@ -9,21 +9,34 @@ var EditDashboardModal = React.createClass({
         return {
             id: this.props.id,
             name: this.props.name,
-            pattern: this.props.pattern
+            pattern: this.props.pattern,
+            _error: false,
+            _error_message: ""
```

Comment:
You can remove the `_` prefix from error and error message, in the end is the component status and it's private :)